locales/templates/app.pot

```diff
@@ -1556,77 +1556,617 @@ msgid ""
-#. %1$s will be replaced with a smiling face with smiling eyes Emoji (U+1F60A)
+#. MSRP Vertical
+#. Logoman
+#. The %1$s here is a placeholder for "dollar amount", %2$s here is a
+#. placeholder for "shopping site"
 #, c-format
-msgctxt "exp_survey_thanks"
-msgid "Thank you for giving us feedback %1$s"
+msgctxt "msrp_logoman_message"
+msgid "Win %1$s %2$s voucher"
 msgstr ""
 
-#. Text on survey feedback button. This means the user want to have this
-#. feature.
-msgctxt "exp_survey_yes"
-msgid "YES, PLEASE"
+#. Contextual hint
+#. The %1$s here is a placeholder for "dollar amount", %2$s here is a
+#. placeholder for "shopping site"
+#, c-format
+msgctxt "msrp_home_hint"
+msgid "Try Shopping, News, Game everyday to win %1$s %2$s voucher"
 msgstr ""
 
-#. Text on survey feedback button. This means the user don't want this
-#. feature. "Nope" is an informal way to express "No".
-msgctxt "exp_survey_no"
-msgid "NOPE"
+#. Reward
+msgctxt "msrp_reward_title"
+msgid "Reward"
 msgstr ""
 
-#. Title on web auth dialog. This dialog hint user authentication is required
-#. to proceed to the website.
-msgctxt "http_auth_dialog_title"
-msgid "Authentication required"
+msgctxt "msrp_reward_category_1"
+msgid "Mission"
 msgstr ""
 
-#. Text on an input username field in web auth dialog. This hint user to input
-#. username here.
-msgctxt "http_auth_dialog_user_name"
-msgid "Username"
+msgctxt "msrp_reward_category_2"
+msgid "Gift"
 msgstr ""
 
-#. Text on an input password field in web auth dialog. This hint user to input
-#. password here.
-msgctxt "http_auth_dialog_password"
-msgid "Password"
+msgctxt "msrp_reward_challenge_title"
+msgid "Try Shopping, News, Game for 7 days in a row!"
 msgstr ""
 
-#. Name of low priority notification channel, this will be shown on the app
-#. info page in the system setting
-msgctxt "low_priority_notification_channel"
-msgid "Hidden notifications"
+#. The %1$s here is a placeholder for "date"
+#, c-format
+msgctxt "msrp_reward_challenge_expire"
+msgid "Expires %1$s"
+msgstr ""
+
+#. Challenge
+#. The %1$s here is a placeholder for "dollar amount", %2$s here is a
+#. placeholder for "shopping site"
+#, c-format
+msgctxt "msrp_challenge_title"
+msgid "7-day challenge for %1$s %2$s voucher"
+msgstr ""
+
+msgctxt "msrp_challenge_details_title_1"
+msgid "1. Join the 7-Day challenge"
+msgstr ""
+
+#. The %1$s here is a placeholder for "Firefox Lite"
+#, c-format
+msgctxt "msrp_challenge_details_body_1"
+msgid ""
+"Try Shopping, News, Game on %1$s Home by simply clicking the buttons for "
+"seven days in a row."
+msgstr ""
+
+msgctxt "msrp_challenge_details_title_2"
+msgid "2. Use your Firefox account to redeem your reward"
+msgstr ""
+
+msgctxt "msrp_challenge_details_body_2"
+msgid ""
+"Enter your email address to get started. Make sure you sign in to the "
+"account you want to use before you complete your redemption."
+msgstr ""
+
+msgctxt "msrp_challenge_button_1"
```

Comment:
Please add comment here

Reply:
This is a button where users can press to join the reward program

Reply:
We need the  comment as a localization note next to the string